test/e2e/lib/gutenberg/blocks/slideshow-block-component.js

```diff
@@ -12,6 +12,21 @@ class SlideshowBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'Slideshow';
 	static blockName = 'jetpack/slideshow';
 	static blockFrontendSelector = By.css( '.entry-content .wp-block-jetpack-slideshow' );
+
+	/**
+	 * Uploads images to the slideshow.
+	 *
+	 * @param {{imageName: string, fileName: string, file: string}} filesDetails a list of fileDetails
+	 */
+	async uploadImages( filesDetails ) {
+		const fileInputSelector = By.css( `div[id="${ this.blockID.slice( 1 ) }"] input[type=file]` );
```

Comment:
`div[id='${ this.blockID.slice( 1 ) }']` can be just `this.blockID` (see [this comment](https://github.com/Automattic/wp-calypso/pull/45796#discussion_r500856393)).